Assignment_3/.classpath

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file. We discussed earlier that IDE specific files should not be part of the repository.

Reply:
Okay, Sir. The file is removed